### PR TITLE
Add new security recommendation (add new external package to the project

### PR DESCRIPTION
**Pull request by helen-laktionova**
## PR Description

@lenadoc , @andriitserkovnyi the Security team checked the PR. 
Please merge it 

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    link: docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices.html
+---
+
+## How to add new external package to your project
+
+Sometimes you would like to add a new external package to your project. In this case check our doc to make sure that your project is secure.
+
+### Check security (now and future)
+
+1. Check that package has no known security vulnarabilities.
```
Comment (lapin-sergej) [inline docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md:26]: Check that the package has no known security vulnerabilities.

File: docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+## How to add new external package to your project
+
+Sometimes you would like to add a new external package to your project. In this case check our doc to make sure that your project is secure.
+
+### Check security (now and future)
+
+1. Check that package has no known security vulnarabilities. 
+2. Add automated tool to your CI/CD to check it regularly for all the packages.
```
Comment (lapin-sergej) [inline docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md:27]: Add an automated SAST&SCA tool to your CI/CD to check it regularly for all the packages.

File: docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Check with your Legal (now and future)
+
+Every package comes with a licence. 
+1. Check with your legal team that you can use the package with its licence.
+2. Add automated tool to your CI/CD to check it regularly for all the packages.
+
+We can recommend [Snyc](https://snyk.io/) for all the stated above actions. But you are free to use any other tool.
```
[the remark this answers is not in the document]
Comment (helen-laktionova) [reply docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md:35]: Fixed! Thank you!

File: docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md (hunk 2 above)
Comment (helen-laktionova) [reply docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md:27]: Fixed! Thank you!

File: docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md (hunk 1 above)
Comment (helen-laktionova) [reply docs/scos/dev/guidelines/coding-guidelines/secure-coding-practices-external-packages.md:26]: Fixed! Thank you!

**Pull request merged**